tests/test_converters.py

```diff
@@ -649,6 +649,9 @@ def test_format_2_converter(self):
             numpy.vstack([self.f2_train_targets_mock,
                           self.f2_test_targets_mock,
                           self.f2_extra_targets_mock]))
+        assert h5file['targets'][...].max() < 10
```

Comment:
The tests are currently failing because `self.f2_{train,test,extra}_targets_mock` contains labels ranging from 1 to 10.

The following should solve it:

```python
f2_train_targets_mock = self.f2_train_targets_mock
f2_train_targets_mock[f2_train_targets_mock == 10] = 0
f2_test_targets_mock = self.f2_test_targets_mock
f2_test_targets_mock[f2_test_targets_mock == 10] = 0
f2_extra_targets_mock = self.f2_extra_targets_mock
f2_extra_targets_mock[f2_extra_targets_mock == 10] = 0
assert_equal(
    h5file['targets'][...],
    numpy.vstack([f2_train_targets_mock,
                  f2_test_targets_mock,
                  f2_extra_targets_mock]))
```